src/components/Calendar.js

```diff
@@ -45,21 +45,19 @@ const Calendar = ({calendarData}) => {
   const periodRows =  _.keys(data.periods).map((p) => {
     const period = data.periods[p];
     const filteredPeriods = classesByPeriod(data.mySchedule, p);
-    return _.keys(filteredPeriods).map((k) => {
-      return (
+    return (
         <PeriodRow
-          key={k} 
```

Comment:
Absolutely not. We will use the object keys, not names, for the keys. Names can be the same. Object keys can't.